app/views/service_providers/_form.html.erb

```diff
@@ -73,6 +73,20 @@
     <%= form.input :production_issuer,
                    label: 'Production Issuer',
                    input_html: { class: 'usa-input' } %>
+
+    <%# allow_prompt_login checkbox %>
```

Comment:
Should we display a message "Prompt=login enabled" message for non-admin users after prompt=login has been enabled?

Reply:
Good question. I followed the example set with other admin-only attributes and made the display on the show view admin-only as well. I'll add a separate branch for users and we'll only show it if it's turned on.